guide/ops/requirements.md

```diff
@@ -64,14 +64,14 @@ rotation according to your organisation's policy.
 
 ## Supported Operating Systems
 
-The recommended operating system is CentOS 6.x or RedHat 6.x.
+The recommended operating system is CentOS 7.x or RedHat 7.x.
 
-Brooklyn has also been tested on Ubuntu 14.04 and OS X.
+Brooklyn has also been tested on CentOS 6.x, Ubuntu 14.04, Ubuntu 16.04 and OS X.
```

Comment:
Interesting. I think we need three levels then: recommented; tested; and "some folk also use...". We don't want to promise that it will work on all these things if we don't test it on all of them thoroughly every release. Only a subset of the testing is done by people running Windows.